code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -817,7 +817,8 @@
 		return (GLOB.cameranet && GLOB.cameranet.checkTurfVis(get_turf_pixel(A))) || apc_override
 	//AI is carded/shunted
 	//view(src) returns nothing for carded/shunted AIs and they have x-ray vision so just use get_dist
-	return get_dist(src, A) <= client.view
+	var/list/viewscale = getviewsize(client.view)

```

Comment:
Travis is throwing an error because the branch this mirror was made on doesn't have /code/_HELPERS/view.dm included in the .dme. It is included in our base branch however, so it should work.